### PR TITLE
Regenerated and tweaked

### PR DESCRIPTION
**Pull request by fearthecowboy**

### CODE REVIEW

File: schemas/2014-04-01-preview/deploymentTemplate.json [hunk 1]
```diff
@@ -21,223 +21,583 @@
...
-                  { "$ref": "http://schema.management.azure.com/schemas/2017-03-01/Microsoft.Network.json#/resourceDefinitions/loadBalancers" },
-                  { "$ref": "http://schema.management.azure.com/schemas/2017-03-01/Microsoft.Network.json#/resourceDefinitions/networkSecurityGroups" },
-                  { "$ref": "http://schema.management.azure.com/schemas/2017-03-01/Microsoft.Network.json#/resourceDefinitions/networkInterfaces" },
-                  { "$ref": "http://schema.management.azure.com/schemas/2017-03-01/Microsoft.Network.json#/resourceDefinitions/routeTables" },
-                  { "$ref": "http://schema.management.azure.com/schemas/2017-04-01/Microsoft.Insights.json#/resourceDefinitions/actionGroups" },
-                  { "$ref": "http://schema.management.azure.com/schemas/2017-04-01/Microsoft.Insights.json#/resourceDefinitions/activityLogAlerts" },
-                  { "$ref": "#/definitions/resourceGeneric" }                
+                  { "$ref": "#/definitions/resourceGeneric"},
+                  { "$ref": "../2014-02-26/microsoft.visualstudio.json#/resourceDefinitions/account"},
```
Comment (TianoMS) [inline schemas/2014-04-01-preview/deploymentTemplate.json]: >.. [](start = 29, length = 2)

any reason for this change?

File: schemas/2014-12-01/Microsoft.MobileEngagement.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "https://schema.management.azure.com/schemas/2014-12-01/Microsoft.MobileEngagement.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.MobileEngagement",
+  "description": "Microsoft MobileEngagement Resource Types"
+}
```
Comment (TianoMS) [inline schemas/2014-12-01/Microsoft.MobileEngagement.json]: what is this for?

**Comment by JasonYang-MSFT** [comment]
@fearthecowboy @TianoMS 
Any updates on this PR?

**Comment by fearthecowboy** [comment]
This was a regeneration of all the schemas that we've got in the azure-rest-api-specs repository -- in an attempt to get this to fully automated.

Given that adding in all the resources and everything kills perf and exposes flaws in the use of schema itself, this PR is dead.

**Pull request closed without merging**